src/db/fn/listUsersWhere.ts

```diff
@@ -38,13 +28,23 @@ const listUsersWhere =
     location?: string;
     languages?: string[];
   }) => {
-    let sql = ['SELECT * FROM users WHERE 1=1'];
-    let values: string[] = [];
-    ({ sql, values } = addLocationCondition(sql, values, location));
-    ({ sql, values } = addLanguagesCondition(sql, values, languages));
+    let sql = [
+      'SELECT DISTINCT ON (u.login) u.login, u.name,',
+      'u.public_repos, u.followers, u.location,',
+      'ARRAY_AGG(l.name) AS languages',
+      'FROM users u',
+      'LEFT OUTER JOIN users_languages ul ON ul.user_id = u.id',
+      'LEFT OUTER JOIN languages l ON l.id = ul.language_id',
+      'WHERE 1=1',
+    ];
+    sql = addLocationCondition(sql, location);
+    sql = addLanguagesCondition(sql, languages);
+    sql.push(
+      'GROUP BY u.login, u.name, u.public_repos, u.followers, u.location'
+    );
+    sql.push('ORDER BY u.login');
 
-    const query = pgFormat(sql.join(' '), values);
-    return client.query<UserSchema>(query);
+    return client.any<UserSchema>(sql.join(' '));
```

Comment:
Type this better (languages is missing)

Reply:
https://github.com/matteolc/lovelystay/commit/a8bbed1a0441f77ea22bfbc75f4f0a546ceed51d